src/app/cli.sh

```diff
@@ -122,6 +128,7 @@ cli::help() {
 	echo "    horde command [options]"
 	echo
 	echo "COMMANDS:"
+	echo "    init driver_name             use driver_name.initializer.sh to initialize this directory"
```

Comment:
match help style used elsewhere
```suggestion
	echo "    init [driver_name]             use driver_name.initializer.sh to initialize this directory"
```